timemachine/potentials/jax_utils.py

```diff
@@ -1,34 +1,30 @@
-from typing import Tuple
-
-import jax
 import jax.numpy as np
 import numpy as onp
 from jax import vmap
 
 Array = onp.array
```

Comment:
nit: Since you are changing the usage, might be good to switch this to `onp.typing.NDArray`?

Reply:
Updated in https://github.com/proteneer/timemachine/pull/685/commits/78ac174fbfe41e6fccf86917a06aa8c022ae9d7b